content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,111 @@
+# Using CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration (CI) systems (with exceptions as
+noted!).
+
+1. Fork our
+   [example project repository](https://gitlab.com/iterative.ai/example_cml).
+   Click on Fork and select the namespace where you would like to keep the
+   project.
+
+   ![](/img/gitlab_fork_cml_project.png)
+
+2. ⚠️ In GitLab, to use CML, you must create a variable called a repo_token
+   whose value is a Personal Access Token. To do this:
+
+   a. Click on your Avatar in the upper right side and click on Edit Profile.
```

Comment:
It looks like, for consistency, "Edit Profile" should be in quotes like the other buttons are?